src/riak_kv_hll.erl

```diff
@@ -189,7 +189,10 @@ to_version(_Version, HllSet) ->
 stats(HllSet) ->
     [{S, stat(S, HllSet)} || S <- ?STATS].
 
--spec stat(stat(), hllset()) -> byte() | precision() | undefined.
+-spec stat(stat(), hllset()) -> number() | undefined.
```

Comment:
Actually, if I try to make the type:
`-spec stat(stat(), hllset()) -> 1..255 | 4..16 | undefined.` it passes dialyzer

Reply:
Interesting, this is strange behaviour.  I will amend and use that definition